lib/Notification/Notifier.php

```diff
@@ -270,6 +280,44 @@ protected function shortenJsonEncodedMultibyteSave(string $subject, int $dataLen
 		return $temp;
 	}
 
+	/**
+	 * @throws HintException
+	 */
+	protected function parseRemoteInvitationMessage(INotification $notification, IL10N $l): INotification {
+		$subjectParameters = $notification->getSubjectParameters();
+
+		[$sharedById, $sharedByServer] = $this->addressHandler->splitUserRemote($subjectParameters['sharedByFederatedId']);
+
+		$message = $l->t('{user1} shared room {roomName} on {remoteServer} with you');
+		$parsedMessage = $l->t('{user1} shared room {roomName} on {remoteServer} with you', [
```

Comment:
Maybe do something like https://github.com/nextcloud/spreed/blob/928e047de2dca944ba2d1747bd1e3a83e57e746c/lib/Notification/Notifier.php#L425-L432
To avoid the 2 strings or parameter data to divert in the future by accident

Reply:
The original reason was to have the federated ID instead of display name if there is no support for ROS but that should not be too much of an issue.